site/en/blog/how-fugu-is-my-browser-and-how-fugu-is-the-web/index.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: 'layouts/blog-post.njk'
+title: '"How Fugu is my browser?" and "How Fugu is the web?"'
+subhead: >
+  To answer these questions, check out a browser testing site and a browser extension, both focused
+  on the web's capabilities.
+date: 2023-01-16
+# updated: 2023-01-16
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/8FZcBmFowbDKWxpkOytx.jpg
+alt: Blowfish swarm swimming in the ocean.
+authors:
+  - thomassteiner
+tags:
+  - capabilities
+---
+
+## How Fugu is my browser?
```

Comment:
We're starting with an immediate subhead here. Can we introduce this article (I think your title isn't helping you with the structure.)